x-pack/solutions/observability/plugins/observability/server/routes/alerts/route.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  getRelatedDashboardsParamsSchema,
+  GetRelatedDashboardsResponse,
+} from '@kbn/observability-schema';
+import { IKibanaResponse } from '@kbn/core-http-server';
+import type { SavedObjectsFindResult } from '@kbn/core/server';
+import type { DashboardAttributes } from '@kbn/dashboard-plugin/server/content_management/v3';
+import { createObservabilityServerRoute } from '../create_observability_server_route';
+import { SuggestedDashboardsClient } from '../../services/suggested_dashboards_client';
+import { InvestigateAlertsClient } from '../../services/investigate_alerts_client';
+import { AlertNotFoundError } from '../../common/errors/alert_not_found_error';
+
+const alertsDynamicDashboardSuggestions = createObservabilityServerRoute({
+  endpoint: 'GET /internal/observability/alerts/suggested_dashboards',
```

Comment:
This endpoint returns both suggested and linked dashboards. Let's rename to `related_dashboards`